src/crypto/store/indexeddb-crypto-store-backend.js

```diff
@@ -60,6 +60,7 @@ export class Backend {
         return new Promise((resolve, reject) => {
             const txn = this._db.transaction("outgoingRoomKeyRequests", "readwrite");
             txn.onerror = reject;
+            txn.onabort = (ev) => reject(ev.target.error);
```

Comment:
Might be missing something, but looks like you could use `promiseifyTxn` here.

Reply:
yeah okay

Reply:
nope, because we need to do another call first and need to get a handle on the `resolve` in linen 73